app/assets/javascripts/map.js

```diff
@@ -1,3 +1,12 @@
+
+// javascript for expander
+$(document).ready(function() {
+  $('.expander-trigger').click(function(){
```

Comment:
Lines 4-6 can be in the function on line 14 instead of line 3. The `$(document).reach(function() {` and `$(function() {` do the same thing, so in this case they are redundant. Can you move lines 4-6 into the `$(function() {` on line 14?